cmd/kubermatic-installer/shared.go

```diff
@@ -26,11 +27,13 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 
+	kubermaticv1 "k8c.io/kubermatic/v2/pkg/apis/kubermatic/v1"
 	operatorv1alpha1 "k8c.io/kubermatic/v2/pkg/crd/operator/v1alpha1"
```

Comment:
I thought we moved operator APIs too

Reply:
We did. But the installer needs to migrate the old config, so it needs the old types.